.github/workflows/package-apisix-openresty.yml

```diff
@@ -0,0 +1,44 @@
+name: package apisix-openresty
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      BUILD_APISIX_OR_VERSION: 1.0.0
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: install dependencies
+        run: |
+          sudo apt-get install -y make ruby ruby-dev rubygems build-essential
+          sudo gem install --no-document fpm
+          sudo apt-get install -y rpm
+
+      - name: build apisix-openresty rpm
+        run: |
+          make package type=rpm app=apisix-openresty version=${BUILD_APISIX_OR_VERSION}
+
+      - name: run centos7 docker and mapping apisix-openresty rpm into container
+        run: |
+          docker run -itd -v /home/runner/work/apisix-build-tools/apisix-build-tools/output:/output --name centos7Instance --net="host" docker.io/centos:7 /bin/bash
+
+      - name: install dependencies in container
+        run: |
+          docker exec centos7Instance bash -c "yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo"
+          docker exec centos7Instance bash -c "yum -y install openresty-openssl111 openresty-pcre openresty-zlib"
+
+      - name: install rpm in container
+        run: |
+          docker exec centos7Instance bash -c "yum -y localinstall /output/apisix-openresty-${BUILD_APISIX_OR_VERSION}-0.x86_64.rpm"
```

Comment:
do we need to add some checks here on whether installation succeeds

Reply:
yes, has added, take a look at below task: `check and ensure openresty is installed`

Reply:
Ah get it